src/main/java/no/nav/familie/integrasjoner/personopplysning/PersonopplysningerService.kt

```diff
@@ -98,6 +100,12 @@ class PersonopplysningerService(private val personSoapClient: PersonSoapClient,
                                    barnsForeldrer = barnsForeldrer)
     }
 
+    @Cacheable(cacheNames = ["ADRESSEBESKYTTELSE"], key = "#personIdent + #tema", condition = "#personIdent != null")
```

Comment:
Tror ikke det er lov å cache adressebeskyttelse mer enn en time, hvor lenge caches det her?

Reply:
Ok! Det var ny info for meg, men gir vel også mening
`.expireAfterWrite(2, TimeUnit.HOURS)` fra cacheManager

Hvis vi endrer denne til 1 hour, eller om vi har en egen cache for det som er koblet til adressebeskyttelse/tilgang så går det fint.

Må bare huske på att klientene(ef-sak/ba-sak) kanskje også burde forholde seg til å ikke cachea denne informasjonen for lenge. Eks hvis man henter den når det bare er 10 min kvar på cachen, og så cacher ef-sak selv i 1h... ?

Reply:
Vi cachear allerede denne informasjonen på et annet sted, men att vi cachear hele personen
Sånn sett så endrer egentlige ikke denne noe fra tidligere, og tenker vi kan rydde opp det i en egen branch

Reply:
Det brukes en generell cache på 2 timer. Der lagrer vi allerede adressebeskyttelse :/ Men det er allerede cache på tilgangskontroll-metodene, så kan ev fjerne cachen på person-metoden?

Reply:
Har bare fjernet cachingen av adressebeskyttelse :)